### DISCUSSION TITLE
Mac Shortcuts

### DISCUSSION THREAD

**Discussion by mzuniga51** [jgm/pandoc#9840] [Q&A]
Hello everyone!  I'm trying to find a way to convert word and ppt docs to pdf's with pandoc.  Ideally there should be a few mac shortcuts or applescripts that can do the following: 
a. Convert one word or ppt document to pdf
b. Batch convert a few word or ppt documents to pdf
c. Combine word documents in one single pdf document
Problem is I'm quite newby to pandoc and I have no idea where to start.  Can someone point me in the right direction?
Thanks in advance to the community.

**Comment by TomBener** [comment]
Here is a shortcut to convert epub to html (the path of pandoc may need to be changed): https://www.icloud.com/shortcuts/af4970950e334663aa45f69c99c96494

You can use it as a start and create your own automation tools. In addition, Keyboard Maestro, Alfred, Raycast can also be used to execute pandoc commands.

**Reply by mzuniga51** [reply]
Many thanks Tom!
Now I have successfully converted your shortcut to convert word to pdf  lookin like this:  https://www.icloud.com/shortcuts/61aef27deb054dacbdeb08a90fe7ebe2
Still no idea how to make it so I can convert several documents in one go or consolidate several word docs on a single pdf. Any help there is highly appreciated!

**Reply by TomBener** [reply]
I have not tested Apple Shortcuts, but you can refer to [this question and answers below](https://apple.stackexchange.com/questions/430283/receiving-multiple-files-not-just-one-in-a-shortcut-on-macos-12-monterey).

I created a Keyboard Maestro macro to convert DOCX files to PDF, which supports multiple input files.

<img width="500" alt="CleanShot 2024-06-04 at 17 26 26@2x" src="https://github.com/jgm/pandoc/assets/49151155/3eed891c-6d75-4e7e-bf3c-d9ad8e9f8724">

You can download it [here](https://share.zight.com/Kouk4YPG).

It is easy to modify this macro to add steps to concatenate converted PDFs into a single one.
